.vitepress/constants/mocha_versions.js

```diff
@@ -1,9 +1,9 @@
 const mochaVersions = Object.freeze({
-  "app-latest-tag": "v1.11.0",
-  "app-latest-sha": "21b5bc747c8500e4888474df7d828e66c33f332d",
-  "core-latest-tag": "v1.36.1-tm-v0.34.29",
-  "core-latest-sha": "624c43d4484a785ec855f5fb93ea571c1e728fda",
-  "node-latest-tag": "v0.14.1",
-  "node-latest-sha": "6549873516889aebcb787c4e9f2ae3b3258c386a",
+  "app-latest-tag": "v2.0.0",
+  "app-latest-sha": "3554ad6608e4028e9f7d5f479a350ab12dc4e314",
+  "core-latest-tag": "v1.38.0-tm-v0.34.29",
+  "core-latest-sha": "637b18e666f995cda84ace6d7000a199b42e208b",
+  "node-latest-tag": "v0.16.0-rc0",
```

Comment:
Note: this line may be updated to `v0.16.0` if node cuts an official release before August 21.

cc: @renaynay